R/standalone-types-check.R

```diff
@@ -460,20 +462,26 @@ check_formula <- function(x,
 check_character <- function(x,
                             ...,
                             allow_null = FALSE,
+                            min_length = 0,
                             arg = caller_arg(x),
                             call = caller_env()) {
   if (!missing(x)) {
-    if (is_character(x)) {
+    if (is_character(x) && (length(x) >= min_length)) {
       return(invisible(NULL))
     }
     if (allow_null && is_null(x)) {
       return(invisible(NULL))
     }
   }
 
+  what <- "a character vector"  
+  if (min_length > 0) {
```

Comment:
```suggestion
  if (min_length > 1) {
```

Actually, the error message will be clear enough for length 1